pokete.py

```diff
@@ -415,12 +415,12 @@ def set_money(self, money):
         """Sets the money to a certain value
         ARGS:
             money: New value"""
-        assert money >= 0, "money has to be positive"
-        logging.info("[Figure] Money set to %d$ from %d$",
+        assert money >= 0, "Money has to be positive."
+        logging.info("[Figure] Money set to $%d from $%d",
                      money, self.__money)
         self.__money = money
         for cls in [inv, buy]:
-            cls.money_label.rechar(str(self.__money) + "$")
+            cls.money_label.rechar("$" + str(self.__money))
```

Comment:
It really depends, but not in most English-speaking countries that I am aware of.

I know that both the US and UK put the currency symbol in front of the number, which I know is counterintuitive because one would say "100 dollars" but write it like they're saying "dollars 100".